app/services/data_stage/fetch_gias_data_files.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "gias_api_client"
+require "csv"
+
+module DataStage
+  class FetchGiasDataFiles < ::BaseService
```

Comment:
Is this used anywhere?

Reply:
Yes, it fetches the GIAS files. At the moment there is no job set up to do this automatically but in the Guidance to review bit above I have an example of how it is used